nms/packages/magmalte/app/components/context/SubscriberContext.js

```diff
@@ -39,6 +40,7 @@ setState: POST, PUT, DELETE subscriber
 export type SubscriberContextType = {
   state: {[string]: subscriber},
   sessionState: {[string]: subscriber_state},
+  forbiddenNetworkType: {[string]: core_network_types},
```

Comment:
This should definitely be plural.
```suggestion
  forbiddenNetworkTypes: {[string]: core_network_types},
```

Reply:
Done.